src/site/content/en/handbook/effective-instruction/index.md

```diff
@@ -5,22 +5,22 @@ authors:
   - mfriesenhahn
 date: 2019-06-26
 description: |
-  Strategies for writing effective instruction on web.dev, especially for adult learners.
+  Learn strategies for writing effective instruction on web.dev, especially for adult learners.
 ---
 
 Most web.dev content includes at least some instruction. These guidelines help make your instruction easy to follow.
 
-Explain _why_ to do something before explaining _how_ to do it. Knowing what they're trying to accomplish helps readers understand the purpose of individual steps.
+Explain _why_ to do something before explaining _how_ to do it. Knowing the end goal helps readers understand the purpose of individual steps.
 
 Keep the scope of instruction manageable. If a task is complex, find ways to chunk the steps. For example, provide the essential pieces in a post and put the ancillary pieces in one or more codelabs.
 
-Always provide examples for complex or abstract concepts. If you're debating whether to give an example, do.
+Always provide examples for complex or abstract concepts. If you're debating whether to give an example, add one.
 
-Use web.dev's built-in [Instruction components](/handbook/web-dev-components#instruction) for instructing common tasks in the tools often referenced on the site (e.g., Glitch, Chrome DevTools). This approach provides a consistent user experience and makes it easy to update instructions across the site when needed.
+Use web.dev's built-in [Instruction components](/handbook/web-dev-components#instruction) for common tasks that are often referenced on the site (for example, instructions about opening Glitch or Chrome DevTools). This approach provides a consistent user experience and makes it easy to update instructions across the site when needed.
 
 ## Instruction for adult learners
-Adult learners bring a lot experience. It's important to honor that experience while acknowledging the challenges of the tasks you're teaching.
+Adult learners bring a lot experience. It's important to honor that experience while acknowledging the challenges of the tasks you're teaching. 
 
-Be mindful of [voice](/handbook/voice). Try to strike a balance between showing expertise and humility to avoid alienating your readers.
+Be mindful of [voice](/handbook/voice). To avoid alienating your readers, try to strike a balance between showing expertise and humility.
```

Comment:
Is the balance between expertise/humility covered in voice? This is such a short sentence, that I'm almost not sure how to do it. Examples would be helpful so I don't have to jump off to a link to find out what this means

Reply:
Yeah, agree about needing examples and non-examples. I'm thinking it might be good to ask tech writers to be on the lookout while they're reviewing content so the examples can be authentic.